LEGO1/realtime/orientableroi.h

```diff
@@ -16,34 +16,29 @@ class OrientableROI : public ROI {
 		ZEROVEC3(m_world_velocity);
 		IDENTMAT4(m_local2world.GetMatrix());
 	}
-	// SYNTHETIC: LEGO1 0x100a4630
-	// OrientableROI::`scalar deleting destructor'
 
-	virtual const Vector3& GetWorldVelocity() const;
-	virtual const BoundingBox& GetWorldBoundingBox() const;
-	virtual const BoundingSphere& GetWorldBoundingSphere() const;
-
-protected:
-	// vtable + 0x14
-	virtual void VTable0x14() { VTable0x1c(); }
-	virtual void UpdateWorldBoundingVolumes() = 0;
+	virtual const Vector3& GetWorldVelocity() const;                // vtable+0x8
+	virtual const BoundingBox& GetWorldBoundingBox() const;         // vtable+0xc
+	virtual const BoundingSphere& GetWorldBoundingSphere() const;   // vtable+0x10
+	virtual void VTable0x14() { VTable0x1c(); }                     // vtable+0x14
```

Comment:
Since it's a virtual function with implementation it should have a function annotation?